.github/workflows/issue_title_check.yaml

```
@@ -0,0 +1,42 @@
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  check_title_length:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check issue title length
+        id: title_check
+        run: |
```

actionlint (1.7.4)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)